doc/tools/docgram/README.md

```diff
@@ -58,22 +58,26 @@ tactic_expr5: [
 2. The tool applies grammar editing operations specified by `common.edit_mlg` to
 generate `editedGrammar`.
 
-3. `orderedGrammar` gives the desired order for the nonterminals and productions
-in the documented grammar.  Developers should edit this file to change the order.
+3. `orderedGrammar` gives the desired order for nonterminals and individual productions
+in the documented grammar.  Developers should edit this file only to reorder lines.
 `doc_grammar` updates `orderedGrammar` so it has the same set of nonterminals and productions
-as `editedGrammar`.  The update process removes manually-added comments from
-`orderedGrammar` while automatically-generated comments will be regenerated.
-
-4. The tool applies further edits to the grammar specified by `productionlist.edit_mlg`,
-then it updates the productionlists in the `.rst` files as specified by comments in the form
-`.. insertgram <first nt> <last nt>`.  The edits are primarily to expand
-`.mlg` constructs such as `LIST1` and `OPT` into separate productions.  The tool
-generates `productionlistGrammar`, which has the entire grammar in the form of `productionlists`.
-
-5. Using the grammar produced in step 3, the tool applies edits specified by
-`prodn.edit_mlg` and generates `prodnGrammar`, representing each production as
-a Sphinx `prodn` construct.  Differently-edited grammars are used because `prodn`
-can naturally represent `LIST1 x SEP ','` whereas that is awkward for `productionlists`.
+as `editedGrammar` while retaining the previous ordering.  Since the position of
+new or renamed nonterminals is unspecified, they tend to show up in the wrong
+place in `orderedGrammar`, therefore users should review the output and make
+appropriate adjustments to the order.
+
+The update process removes manually-added comments from `orderedGrammar` while
```

Comment:
You can indent the content of your bullet-point to avoid new paragraph inside a bullet point to feel oddly unindented in the rendered version.